tests/Unit/ComponentPaginationTest.php

```diff
@@ -48,20 +48,12 @@ public function double_page_value_should_be_casted_to_int()
             ->call('gotoPage', 2.5)
             ->assertSet('page', 2);
     }
-
-    /** @test */
-    public function string_page_value_should_be_reset()
```

Comment:
as i said since cursor pagination use string as identifier we should not reeset strings